ptop.py

```diff
@@ -60,7 +62,8 @@ def ascii_progress_bar(percentage, bar_length=50):
                 lines.extend(new_lines)
 
             clear_console()
-            print(ascii_progress_bar(progress(lines, tj)))
+            print(ascii_progress_bar(progress(lines, tj),tj))
```

Comment:
```suggestion
            if total_jobs > 0:
                print(ascii_progress_bar(progress(lines, tj)))
```

Having the condition inside the function that generates the progress bar feels wrong, no? It is makes the responsibility of the `ascii_progress_bar()` function more complicated. Also, the proposed implementation does not provide a return value for the "else" case. Let's move the condition outside, so the reader of the dashboard printing logic can easily see when the bar is printed and when not.